src/Microsoft.FeatureManagement/FeatureVariant.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// The definition for a variant of a feature.
+    /// </summary>
+    public class FeatureVariant
```

Comment:
Perhaps `VariantDefinition` given it's usage in FeatureDefinition and also to distinguish a bit more between this and `Variant` returned by `IVariantFeatureManager`.